lib/gems/pending/util/extensions/miq-benchmark.rb

```diff
@@ -29,6 +29,10 @@ def self.realtime_block(key, &block)
     begin
       ret = realtime_store(hash, key, &block)
       return ret, hash
+    rescue
+      # Don't let timings be lost on exception when there is nobody else to pick them up.
+      $log.info("Exception in realtime_block #{key.inspect} - Timings: #{hash.inspect}") if outermost
```

Comment:
The only other thing I can think of is a top-level accessor that has the "last" value's run... Something like a last_realtime_block_result.  Downside is you would have this thing hanging around forever, but it isn't very big.